.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+dist: xenial
+
+language: java
+
+services:
+  - docker
+
+env:
+  - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+
+# Run before every job
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -y openjdk-8-jdk
+  - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
+  - java -version
+  - uname -a
+  - chmod +x pom.xml
+
+stages:
+  - test
+
+jobs:
+  include:
+    - stage: test
+      name: unitAndFunctionalTests
+      script:
+        - mvn clean install
```

Comment:
What's interesting is that since Travis by default runs the maven clean install before the script to install all dependencies. So basically we run clean install twice with this command. [Read more here](https://docs.travis-ci.com/user/languages/java/#projects-using-maven). I think using "mvn test" is a better option.

Reply:
done